azure-pipelines.yml

```diff
@@ -40,21 +40,14 @@ variables:
 jobs:
 - template: build-pipeline.yml@templates
   parameters:
+    # Tell the build system that we want to pip install tk-toolchain
+    # from the current branch.
+    tk_toolchain_ref: $(Build.SourceBranch)
+    # These extra repositories are required to test some of the command line tools
     additional_repositories:
     - name: tk-framework-shotgunutils
     - name: tk-multi-publish2
     - name: tk-maya
     - name: tk-config-basic
       ref: v1.3.0
     - name: python-api
-
-    extra_test_dependencies:
-      # Replaces tk-toolchain with this branch's copy
-      - "--editable ."
```

Comment:
See, this is where we force installed the local version. Now, the tk_toolchain_ref at the top takes care of installing tk-toolchain once from the right location.